pyrdp/mitm/FakeServer.py

```diff
@@ -0,0 +1,274 @@
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2022
+# Licensed under the GPLv3 or later.
+#
+import os, random, shutil, socket, subprocess, threading, time
+from tkinter import *
+from PIL import Image, ImageTk
+from pyvirtualdisplay import Display
+from pyrdp.logging import log  # TODO: this logs to GLOBAL and not to the session, setup proper logging
```

Comment:
imports should be organized PEP8 style: stdlib first then external dependencies then local

Reply:
I *think* that's solved.